libr/flag/flag.c

```diff
@@ -624,11 +644,13 @@ int main () {
 	r_flag_set (f, "rip", 4, 4);
 
 	i = r_flag_get (f, "rip");
-	if (i) printf ("nRIP: %p %llx\n", i, i->offset);
+	if (i)
+		printf ("nRIP: %p %llx\n", i, i->offset);
```

Comment:
I ran sys/indent and it changed the whole file. I'm nowhere near this function, I can't fix everything because of that.